### PR TITLE
Fixes for new metadata

### PR DESCRIPTION
**Pull request by katxiao**

### CODE REVIEW

**Review by fealho — APPROVED** [review]
Looks good to me! I'd get a second review from someone else though, there were quite a few changes.

File: sdmetrics/reports/utils.py
Comment (Grayangelz) [inline sdmetrics/reports/utils.py:10]: 12

**Pull request merged**
